waiter/src/waiter/process_request.clj

```diff
@@ -555,13 +555,15 @@
                                   (process-backend-response-fn local-usage-agent instance-request-properties descriptor instance request
                                                                reason-map reservation-status-promise confirm-live-connection-with-abort
                                                                request-state-chan response))
+                                (assoc :backend-response-latency-ms (utils/nanos->millis response-elapsed))
```

Comment:
Is there no path from which `response-elapsed` is defined to use here that an exception can be thrown. If not, the `try-catch` wrapping it can be much narrower.

Reply:
Sorry, I don't follow.  Can you explain a bit more?